### ISSUE TITLE
[fd-file-uploader][Core]  buttonAriaLabel does not work and please support buttonAriaLabelledBy

### ISSUE THREAD

**Issue by Meggie886** [SAP/fundamental-ngx#13230]
Is it a bug ?

Yes, it's a bug.

Example(basic example in the document)
https://stackblitz.com/run?file=src%2Fapp%2Ffile-uploader-example.component.html

![Image](https://github.com/user-attachments/assets/a25c35e1-5ed6-4eef-a1f9-a7f72b14307a)

![Image](https://github.com/user-attachments/assets/10980209-f3f2-42fb-8732-6a403e1e8f84)

For support  buttonAriaLabelledBy, according to ACC-264.1(Level A)

ACC-264.1: Are all UI elements presented correctly using assistive technologies?
Follow the test case.
Identify all [elements on the screen] that are interactive or/and have a semantic meaning or serve only decoration purposes.
Check if [element role, name, state(s), values and other element attributes] for interactive and semantic/non-semantic elements can be determined by [assistive technology (screen reader)]when they have keyboard or virtual focus on desktop devices or been navigated using explore by touch navigation on mobile devices.
